manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml

```
@@ -0,0 +1,63 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Max.Backup
+PackageVersion: 0.1.3
+PackageLocale: en-US
+Publisher: Kelly Elton
+PublisherUrl: https://github.com/kellyelton
+PublisherSupportUrl: https://github.com/kellyelton/MaxBackup/issues
+Author: Kelly Elton
+PackageName: max backup

```

Should this be capitalized and joined?
```suggestion
PackageName: MaxBackup
```

---

is that required?

---

I've made the requested change.